src/features/user.js

```diff
@@ -32,8 +32,12 @@ const userSlice = createSlice({
       if (!state.attrib) return
       state.attrib = { ...state.attrib, ...action.payload }
     },
+    setUserData: (state, action) => {
```

Comment:
Do we need a new action here? Could we not just have used `onProfileUpdate` as it's exactly the same? Maybe change to name to make it more generic, but probably fine.

https://redux.js.org/style-guide/#model-actions-as-events-not-setters

Reply:
AppHeader where developer mode switch is located uses two different containers for user data - `data` object and `attribs` object. I thought `onProfileUpdate` reducer is different to reflect this structure as it updates `attribs`, not `data` and that we want to have them separated like this (including reducers)...

I have changed the name to updateUserData...

<img width="310" alt="image" src="https://github.com/ynput/ayon-frontend/assets/16327908/d470fae1-8422-4f17-a973-caea9f497043">

Reply:
looking at the guide from redux, maybe `userDataUpdate` and `userAttribsUpdate` would be better ? But maybe Im overthinking it...

Reply:
It's not about what data we are updating, it's about what event is causing the update. If onProfileUpdate and onMyUserUpdate update different data then it is correct to seperate them out into two different actions because they are different events that update different data.

But let's stay consistent with the naming convention and rename `updateUserData` to something like `onProfileUpdate`.

> we recommend trying to treat actions more as "describing events that occurred", rather than "setters"

(I know we have setSomeData naming all over actions, but that's bad and we should try to avoid it if we can)